sealer/onstart/onstart.go

```diff
@@ -0,0 +1,36 @@
+package onstart
+
+import (
+	"fmt"
+
+	"github.com/alibaba/sealer/common"
+	"github.com/alibaba/sealer/utils"
+
+	"github.com/alibaba/sealer/image/store"
+)
+
+func initRootDirectory() error {
+	err := utils.MkDirIfNotExists(common.DefaultImageRootDir)
+	if err != nil {
+		return fmt.Errorf("failed to mk default sealer root dir, err: %s", err)
+	}
+
+	err = utils.MkDirIfNotExists(common.DefaultImageMetaRootDir)
+	if err != nil {
+		return fmt.Errorf("failed to mk sealer metadata root dir, err: %s", err)
+	}
+	return nil
+}
+
+func OnBoot() error {
+	err := initRootDirectory()
```

Comment:
The actions of theses functions are similar, maybe we could create an array which saves names of the files to be created and we range the array to create them.